motors_weg_cvw300.orogen

```diff
@@ -71,6 +71,9 @@ task_context "Task", subclasses: "iodrivers_base::Task" do
     property "registers_configuration",
              "/std/vector</motors_weg_cvw300/configuration/MotorRegister>"
 
+    # Makes the fault_state port report the state periodically
+    property "fault_state_periodical", "bool"
```

Comment:
`periodical` would hint at a `period`.

If you want to be verbose, that would be `write_fault_state_at_each_update`

But then, since the GPIOs already have a `edge_triggered_output` property, I think that `edge_triggered_fault_state_output` property would be appropriate (witha  default of true of course)